### PR TITLE
Create liveblocks app updates for integrations

### PR DESCRIPTION
**Pull request by CTNicholas**
Updates for `create-liveblocks-app`. Allows for examples and the starter kit to be deployed using the General and Vercel integrations. The General integration gets the Liveblocks API keys without using Vercel. There are corresponding updates to the starter kit guide.

**TODO**
- [x] Code ready for review @GuillaumeSalles 
- [x] Ready for npm publish (if you're happy) @GuillaumeSalles 
- [x] Docs ready for review
- [x] Merge media: https://github.com/liveblocks/liveblocks.io/pull/851
- [x] Final installer changes
- [ ] Merge after installer published

**How it works**
If you deploy to Vercel, or select "get Liveblocks sercret key automatically", a localhost server starts up, and a browser window opens. When you get to the integration page on liveblocks.io, this page then POSTs the result to the localhost server from the browser. This means you can get your secret key on your local system, and add it to `.env.local`

**Media**
For a quick preview, these are the two videos added to the guide:

https://user-images.githubusercontent.com/33033422/216106789-cc86cbf2-16f7-4f3a-afbe-11ec5b86842e.mp4

https://user-images.githubusercontent.com/33033422/216106813-461f61c8-27d1-470e-8ec7-bf89cb41dc62.mp4

### CODE REVIEW

**Review by adigau — APPROVED** [review]

File: docs/pages/guides/nextjs-starter-kit.mdx [hunk 1]
```diff
@@ -30,104 +30,42 @@ The Next.js Starter Kit includes the following
...
-   root level in your project. This file will contain a randomly generated `NEXTAUTH_SECRET` that will be used for authentication, along with a number of empty variables for you to fill in.
-7. Place your Liveblocks secret key into the file as `LIVEBLOCKS_SECRET_KEY`.
-
-Your `.env.local` file should contain a line similar to this:
-
-```dotenv file=".env.local"
-LIVEBLOCKS_SECRET_KEY =
-  sk_dev_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX;
-```
+Give your project a name, and select the [authentication](#authentication) method you’d like to use in your app.
```
Comment (taylorpwhipp) [inline docs/pages/guides/nextjs-starter-kit.mdx:52]: Suggesting replacing lines 52-66 with this block (you cant make a suggestion over deleted lines that is why I am mentioning this here).    I'd also consider adding a video for Vercel deployment but that is your call @CTNicholas. attaching one to main PR if you think it makes sense lmk and i'll add it.
```suggestion
Give your project a name, and select the [authentication](#authentication) method you’d like to use in your app.
If you are trying to generate a proof of concept quickly, opting to use demo authentication may be a great option- and you can still add other authentication providers later on.

#### Deploy with Vercel Integration
If you would like to set up CICD with your application, we've made that process straightforward for you as well.  When prompted, you can select "deploy to Vercel" to enable building and deploying the starter kit. The Vercel Integration will open in a new browser window for you to complete the process of adding your repository. If by clicking "Create" you recieve "An unexpected internal error occurred" you should validate within your code hosting platofrom of choice (Github, GitLab, Bitbucket) that the Vercel integration has permission to access your repository.  In Github, permissions can be found can be found under Settings > Integrations > Vercel. 

#### Connect to Liveblocks and retrieve your secret key
 If you prefer to work locally, you can tell the installer you would not like to "deploy to Vercel."  After declining the deployment option, the installer will prompt adding your Liveblocks Key to the application automatically.  If you forwent deployment or indicated that you would like to get your Liveblocks secret key automatically, the Liveblocks integration page will open in a new browser tab. 

Once the integration page appears, you can sign up/sign in to Liveblocks, create a new project, and import your API key into the installer.

<Figure>
  <video autoPlay loop muted playsInline>
    <source
      src="/images/docs/next-js-starter-kit/create-liveblocks-project.mp4"
      type="video/mp4"
    />
  </video>
</Figure>

After finishing up, check the installer and follow the commands recommended to get you started.
```

**Comment by taylorpwhipp** [comment]
Below is the sample video I would consider using for the vercel integration.  Let me know your thoughts.  I can use as is or record something different if you had something else in mind.
https://user-images.githubusercontent.com/36981344/216541164-911f0e74-904f-4ecb-9102-6ba15c4a6c87.mp4

File: docs/pages/guides/nextjs-starter-kit.mdx (hunk 1 above)
Comment (adigau) [reply docs/pages/guides/nextjs-starter-kit.mdx:52]: CICD should be CI/CD

File: docs/pages/guides/nextjs-starter-kit.mdx [hunk 2]
```diff
@@ -30,104 +30,58 @@ The Next.js Starter Kit includes the following
...
 
-5. From the command line, get the starter kit by running Create Liveblocks App. Follow through the instructions, selecting your project name, chosen authentication method, and install the app:
+Give your project a name, and select the [authentication](#authentication) method you’d like to use in your app.
+If you are trying to generate a proof of concept quickly, opting to use demo authentication may be a great option—you can still add other authentication providers later on.
 
-```shell
-npx create-liveblocks-app --next
-```
+#### Deploy with Vercel Integration
+If you would like to set up CI/CD with your application, we've made that process straightforward for you as well.  When prompted, you can select "deploy to Vercel" to enable building and deploying the starter kit. The Vercel Integration will open in a new browser window for you to complete the process of adding your repository.
```
Comment (adigau) [inline docs/pages/guides/nextjs-starter-kit.mdx:56]:
```suggestion
If you would like to set up CI/CD with your application, we’ve made that process straightforward for you as well. When prompted, you can select "deploy to Vercel" to enable building and deploying the starter kit. The Vercel Integration will open in a new browser window for you to complete the process of adding your repository.
```

Double spaces @taylorpwhipp and curly apostrophe. Can you make they're all checked?

Comment (adigau) [reply docs/pages/guides/nextjs-starter-kit.mdx]: I fixed them to approve the PR

**Review by adigau — APPROVED** [review]

File: docs/pages/guides/nextjs-starter-kit.mdx (hunk 2 above)
Comment (CTNicholas) [reply docs/pages/guides/nextjs-starter-kit.mdx]: Thanks!

**Pull request merged**